docs/sdk/authentication/session-sigs/intro.md

```diff
@@ -11,116 +11,63 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
 
-`SessionSigs` are produced by a ed25519 keypair that is generated randomly on the browser and stored in local storage. The first step to producing `SessionSigs` is to first obtain an `AuthSig` through an authentication method like Google OAuth (example [here](https://github.com/LIT-Protocol/oauth-pkp-signup-example/blob/main/src/App.tsx#L398)). By specifying the session keypair's public key in the signature payload of the `AuthSig` - the `uri` field of the SIWE - users can choose which specific actions to delegate to the session keypair for operating upon certain resources.
+Session signatures are created with session keys, which are generated when you initiate a session through a request to a Lit network using the Lit SDK. Session keys are unique [`ed25519`](https://ed25519.cr.yp.to/) keypairs generated locally by the Lit SDK and are used to sign all requests to the Lit Network during the current session. 
 
-The session keypair is used to sign all requests to the Lit Nodes, and the user's `AuthSig` is sent along with the request, attached as a "capability" to the session signature. Each node in the Lit Network receives a unique signature for each request, and can verify that the user owns the wallet address that signed the capability.
+While session signatures facilitate ongoing communication during a session, an `AuthSig` (Authentication Signature) is used to verify your identity and authorization to the Lit Nodes.
 
-## Capability Objects
+An `AuthSig` is an [ERC-5573](https://eips.ethereum.org/EIPS/eip-5573) Sign-In with Ethereum Capabilities message that specifies the Lit Resources and Lit Abilities of the session.
 
-Session signatures work by having scoped capabilities be granted to session keys by an inner `AuthSig`. The capability object is a [SIWE ReCap](https://eips.ethereum.org/EIPS/eip-5573) object.
+The `AuthSig` allows Lit nodes to verify your authorization for requested actions, such as decrypting data, signing transactions with a PKP, or transferring PKP ownership. This ensures that only authorized users can perform specific actions within the Lit network.
+When you make a request, each Lit Node checks your `AuthSig` to confirm that your request aligns with the capabilities you previously defined. 
 
-Read more [here](capability-objects) on the session capability objects that we use.
+This authentication system enhances the security of the Lit network. For detailed explanations of this setup, please refer to our [Security Considerations](../security.md) page.
 
-## Format of `SessionSigs`
+## SessionSigs Generation Diagram
+![Session Signatures Diagram](../../../../static/img//SessionSigs.png)
 
-Given the following example `AuthSig`:
+## Paying for Usage of the Lit Network
```

Comment:
Is this section needed? The same info gets covered by the Delegation doc page in the Paying for Lit PR

Reply:
I wouldn't say necessary, but maybe helpful to the page? How about I just keep the first two sentences and add a link to the Delegation doc page once its up?

Reply:
Ok, but I say something along the lines of:

You can facilitate payment for usage the Lit network within Session Signatures. You can read more about paying for usage [here], and paying using Session Signatures [here].

Reply:
Will push this linking to the capacity credits pages and update with the proper links once #319 has been merged